lib/pbench/server/database/models/users.py

```diff
@@ -18,6 +18,7 @@ class User(Database.Base):
     password = Column(LargeBinary(128), nullable=False)
     registered_on = Column(DateTime, nullable=False, default=datetime.datetime.now())
     email = Column(String(255), unique=True, nullable=False)
+    entitlement = Column(String(255), unique=False, nullable=True)
```

Comment:
These feels a lot like a "role".  Is there a reason we can't just call it "role" where we only support one role, "administrator"?

Reply:
Are you suggesting changing the column to a boolean and restricting the value to either true/false for administrator? that's one possibility, but having a string column we can quickly adopt to supporting multiple roles if we want to.

Reply:
I think he's suggesting changing the name from "entitlement" to "role"....  :-)